flang/lib/Lower/OpenMP.cpp

```diff
@@ -173,7 +173,7 @@ static void createBodyOfOp(
   if (args.size()) {
     std::uint64_t loopVarTypeSize = 0;
     for (const Fortran::semantics::Symbol *arg : args)
-      loopVarTypeSize = std::max(loopVarTypeSize, arg->GetUltimate().size());
+      loopVarTypeSize = std::max(loopVarTypeSize, static_cast<std::uint64_t>(arg->GetUltimate().size()));
```

Comment:
Why not change the declaration of `loopVarTypeSize` to `std::size_t` instead? This way you could keep the invocation of `std::max` short.

Reply:
Done. Good point.
I was thinking that loopVarTypeSize is used in an MLIR operation which expects a 64 bit quantity. But it does not seem to be so. I have replaced now.